examples/12_linearization/analysis_options_doe.yaml

```diff
@@ -1,5 +1,5 @@
 general:
-    folder_output: outputs/tower_doe
+    folder_output: outputs
```

Comment:
Users will likely change this, but some standardization of outputs/ and the gitignore would be helpful.  Maybe let's revert this for now.